python/cugraph/cugraph/link_analysis/hits.py

```diff
@@ -79,6 +80,9 @@ def hits(G, max_iter=100, tol=1.0e-5, nstart=None, normalized=True):
     >>> hits = cugraph.hits(G, max_iter = 50)
 
     """
+    warning_msg = ("This call is deprecated and will be refactored "
+                   "in the next release")
+    warnings.warn(warning_msg, PendingDeprecationWarning)
```

Comment:
Reminder that there's no need to have a deprecation warning since you'll just be replacing this entire API with the new one that has the same signature.

Reply:
Yes, I was working on a push right when you added these comments, they've all been addressed in the latest push